### DISCUSSION TITLE
Which version is best to use for openSUSE?

### DISCUSSION THREAD

**Discussion by roke-julian-lockhart** [PowerShell/PowerShell#20872] [Q&A]
Specifically cpe:/o:opensuse:tumbleweed:20231206. https://en.opensuse.org/PowerShell#Installation explains why I ask this.

**Comment by rhubarb-geek-nz** [comment]
Hi, firstly I would not make any recommendation on tumbleweed because it is a moving target.

However I have done some testing on openSUSE Leap 15.5

I tried the one listed in the article..

```
~> sudo zypper install https://github.com/PowerShell/PowerShell/releases/download/v7.2.3/powershell-lts-7.2.3-1.rh.x86_64.rpm
Loading repository data...
Reading installed packages...
Resolving package dependencies...

Problem: nothing provides 'openssl-libs' needed by the to be installed powershell-lts-7.2.3-1.rh.x86_64
 Solution 1: do not install powershell-lts-7.2.3-1.rh.x86_64
 Solution 2: break powershell-lts-7.2.3-1.rh.x86_64 by ignoring some of its dependencies

Choose from above solutions by number or cancel [1/2/c/d/?] (c):
```

I counted that as a no.

I then went through....

```
https://github.com/PowerShell/PowerShell/releases/download/v7.4.0/powershell-7.4.0-1.rh.x86_64.rpm
https://github.com/PowerShell/PowerShell/releases/download/v7.3.10/powershell-7.3.10-1.rh.x86_64.rpm
https://github.com/PowerShell/PowerShell/releases/download/v7.2.17/powershell-7.2.17-1.rh.x86_64.rpm
https://github.com/PowerShell/PowerShell/releases/download/v7.1.7/powershell-7.1.7-1.centos.8.x86_64.rpm
https://github.com/PowerShell/PowerShell/releases/download/v7.1.7/powershell-7.1.7-1.rhel.7.x86_64.rpm
https://github.com/PowerShell/PowerShell/releases/download/v7.0.13/powershell-7.0.13-1.centos.8.x86_64.rpm
https://github.com/PowerShell/PowerShell/releases/download/v7.0.13/powershell-7.0.13-1.rhel.7.x86_64.rpm
```

And they all failed in the same way. 

openSUSE is not binary compatible in the same way that centos/almalinux/rockylinux/rhel and even ol are.

I suggest it will need a specific build for that environment, from dotnet-runtime upwards.

**Reply by roke-julian-lockhart** [reply]
> https://github.com/orgs/PowerShell/discussions/20872#discussioncomment-7803384

Luckily, @rhubarb-geek-nz, I have filed https://github.com/PowerShell/PowerShell/issues/20469#issue-1935587749. I just don't want to be forced to use the snap until that's completed.
